src/conventions/datapack_advancement.md

```diff
@@ -116,3 +116,23 @@ This is an optional syntax that you can take to display an installed datapack *w
 To implement this you simply skip the [#2: Namespace Advancement](#2-namespace-advancement) step and make sure that in step [#3: Datapack Advancement](#3-datapack-advancement) you create your file inside the `/data/global/advancements/standalone/` directory.
 
 > Keep in mind that if you are planning to release more datapacks under this name, you should use the normal syntax over this one.
+
+--------------------
+
+### **Note from Convention Mod (Optional)**
+
+**3. Datapack Advancement**

```

Comment:
Title on lines 124 and 132 appear to be unrelated

Reply:
```suggestion
**Regular format**

```